packages/api/src/chats/usecases/add-message-to-chat.usecase.ts

```diff
@@ -8,6 +8,8 @@ import { Usecase } from '@/common/types/usecase';
 import { ChatMessageResponseSchema } from '@/contract/chats/chat-message.response.dto';
 import { Injectable } from '@nestjs/common';
 
+const TOKENS_LIMIT = 3000;
```

Comment:
this is going to live in our application configurations service. It reads from the `api/config/default.json`. It's going to be another one of the `ai` configurations

Reply:
Yup, done here 22bbeb5acd58edcc08310a9f9035d455423b046a